client/src/components/ExperimentsView.vue

```diff
@@ -114,26 +127,32 @@ export default {
             <li
               v-for="scan of scansForExperiment(experiment.id)"
               :key="`s.${scan.id}`"
-              :class="{
-                current: scan === currentScan
-              }"
-              class="body-1"
+              :class="'body-1 state-'+scanState(scan).replace(/ /g, '-')+scanIsCurrent(scan)"
```

Comment:
I like to minimize the amount of logic in the templates, so it seems cleaner to use a `scanStateClass` method that does something like this:
```
scanStateClass(scan) {
  var classes = `body-1 state-{this.scanState(scan).replace(/ /g, '-')}`;
  if (scan === this.currentScan) {
    classes = `{classes} current`;
  }
  return classes;
}
```